src/pages/index.js

```diff
@@ -3,7 +3,7 @@ import React from 'react'
 import GlobalStyles from '@styles/global'
 import fireDb from '@utilities/firebase'
 import { serialize } from '@utilities/format'
-import FAQExpandable from '@components/faqTemplates/ExpandableWithCategories'
+import FAQExpandable from '@components/faqTemplates/Cmdf2021'
```

Comment:
Any reason for this change?

Reply:
Realized FAQ may not be one of the sections that we can just use a template for. There are too many custom css styling. Might create a node module next year instead. idk :)